documentation/source/users/rmg/input.rst

```diff
@@ -944,3 +944,41 @@ For example ::
 		maxNumSpecies=100
 	)
 
+Restarting from a Seed Mechanism
+=================================
+The preferred method for restarting an RMG-Py job is to restart the job from a seed mechanism. There are many scenarios
+when the user might want to do this, including continuing on a job that ran out of time or crashed as the result of a
+now fixed bug. To restart from a seed mechanism, the block below must be added on to the input file.  ::
+
+    restartFromSeed(path='seed')
+
+The ``path`` flag is the path to the seed mechanism folder, which contains three subfolders that must be titled as
+follows: Filters, Seed, Seed_Edge. The path can be a relative path from the input.py file or an absolute path on disk
+
+Alternatively, you may also specify the paths to each of the following files/directories: coreSeed (path to seed
+mechanism folder containing files named ``reactions.py`` and ```dictionary.txt``` that will go into the model core),
+edgeSeed (path to edge seed mechanism folder containing files named ``reactions.py`` and ```dictionary.txt``` that will
+go into the model edge), filters (an h5 binary file storing the uni-, bi-, and optionally tri-molecular thresholds
+generated by RMG from previous run (must be RMG version >= 3.0)), speciesMap (a YAML file specifying what species are
```

Comment:
Is the v2.5 cutoff going to be before this PR?

Reply:
Up to you. I'll update this based on whatever we decide. I'll wait to push until I hear back on this.

Reply:
As @mjohnson541 suggested we can just say >2.4.1